float32 x
float32 y
float32 theta
float32 v
float32 w
---
float32 x_cord
float32 y_cord
